tests/common/plugins/custom_skipif/CustomSkipIf.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABCMeta, abstractmethod
+
+
+class CustomSkipIf:
+    __metaclass__ = ABCMeta
+    # def __init__(self, ignore_list, extra_params):
+    #     self.name = __name__
+    #     self.ignore_list = ignore_list
+    #     self.extra_params = extra_params
+
+    @abstractmethod
+    def is_skip_required(self, skip_dict_result):
```

Comment:
Suggest having this function simply return a True / False value. 

You could have a separate function which you call to do the parallel processing which calls this one. It would keep your API separate from your implementation and make each implementation of a custom skip class much more readable.

Reply:
In this case we will not be able to see in result - why test skipped.

Reply:
I guess that you would like to return the skip reason, if it is skipped